src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int RANGE_BOUND = 5;
+    public static final int DEFAULT_RADIUS = 10;
+    public static final int MAXIMUM_LENGTH = 50;
+    private Random random = new Random();
+    private Figure figure;
+
+    public Figure getRandomFigure() {
+        int randomInt = random.nextInt(RANGE_BOUND);
+        int randomSideOne = 1 + random.nextInt(MAXIMUM_LENGTH);
```

Comment:
create private method randomSize() and reuse it for creating each type of figure